### ISSUE TITLE
Inconsistent column data: Conversion failed: expected an i32 number in column 'type', found []

### ISSUE THREAD

**Issue by bowencool** [tidbcloud/prisma-adapter#19]
I'm migrating my database from PlanetScale, and an error occurred. It works fine with `@prisma/adapter-planetscale`, so I don't think the issue is related to `@prisma/client`.

```
[serverless-js debug] sql: SELECT ... LIMIT 20 OFFSET 0
[serverless-js debug] request id: 2024031410535853dyre2yh16lhjywmbcm
[serverless-js debug] response id: 2024031410535853dyre2yh16lhjywmbcm
[serverless-js debug] Content-Encoding: gzip
PrismaClientKnownRequestError: 
Invalid `prisma.userBalanceRecord.findMany()` invocation:
Inconsistent column data: Conversion failed: expected an i32 number in column 'type', found []
at _n.handleRequestError (/var/task/node_modules/@prisma/client/runtime/library.js:123:6854)
at _n.handleAndLogRequestError (/var/task/node_modules/@prisma/client/runtime/library.js:123:6188)
at _n.request (/var/task/node_modules/@prisma/client/runtime/library.js:123:5896)
at async l (/var/task/node_modules/@prisma/client/runtime/library.js:128:10871)
at async PayController.getRecord (/var/task/dist/src/pay/pay.controller.js:173:21)
at async /var/task/node_modules/@nestjs/core/router/router-execution-context.js:46:28
at async /var/task/node_modules/@nestjs/core/router/router-proxy.js:9:17
```

``` json
{
    "@prisma/client": "^5.11.0",
    "@tidbcloud/prisma-adapter": "^5.11.0",
    "@tidbcloud/serverless": "^0.1.1",
    "prisma": "^5.11.0",
}
```

The column `type` was defined as `tinyInt`: 
`type         Int      @db.TinyInt`

**Comment by bowencool** [comment]
Adding `fullResult` works for me:

```diff
- const connection = connect({ url: process.env.DATABASE_URL });
+ const connection = connect({ url: process.env.DATABASE_URL, fullResult: true });
```

**Comment by bowencool** [comment]
It seems there were some issues with `tinyint`. This issue was resolved when I changed the column definition from `tinyint` to `int`.

**Comment by shiyuhang0** [comment]
It is a bug and was introduced by a stupid typo.

It was fixed in https://github.com/tidbcloud/prisma-adapter/pull/22 and released as v5.11.1
